README.md

```diff
@@ -26,19 +26,19 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 MATH MAGICIAN <a name="about-project"></a>
+# 📖 BOOK STORE <a name="about-project"></a>
```

Comment:
- [ ] The following sections in your README contain dummy text from the template: Usage, Install, Setup, FAQ (optional - can be removed). Kindly update your sections to be more descriptive.

- [ ] Kindly add the Future Features section in your README to tally with the link in the Table of contents.

- [ ] Your Key Features section still contains information about Math-magician. Kindly fix that.